data.sql

```diff
@@ -1,7 +1,17 @@
 /* Populate database with sample data. */
 
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES (2, 'Gabumon', '2018-11-15', 2, TRUE, 8.0),
-(3, 'Pikachu', '2021-07-01', 1, FALSE, 15.04),
-(4, 'Devimon', '2017-05-17', 5, TRUE, 11);
+VALUES ('Gabumon', '2018-11-15', 2, TRUE, 8.0),
+('Pikachu', '2021-07-01', 1, FALSE, 15.04),
+('Devimon', '2017-05-17', 5, TRUE, 11.0);
+
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
```

Comment:
- [ ] Kindly remove `id` from line 9 also as requested by the previous reviewer.